common/src/main/java/noobanidus/mods/lootr/common/block/entity/BlockEntityTicker.java

```diff
@@ -72,35 +72,26 @@ public static void addEntry(RandomizableContainerBlockEntity incoming, Level lev
 
     Entry newEntry = new Entry(dimension, position, chunks, LootrAPI.getCurrentTicks());
     synchronized (listLock) {
```

Comment:
There is really only one list that needs protection in this class, so I recommend making this 
```suggestion
    synchronized (pendingEntries) {
```
and removing `listLock`